recipe/meta.yaml

```diff
@@ -23,14 +24,15 @@ requirements:
     - setuptools >=60
   # Base from https://github.com/E3SM-Project/zppy-interfaces/blob/v{{ version }}/conda/dev.yml
   run:
-    - python >={{ python_min }}
+    - python >={{ python_min }},<3.14
     - beautifulsoup4
     - lxml
-    - matplotlib-base >=3.8.2,<3.10
+    - matplotlib-base >=3.8.2
     - mpas_tools >=0.21.0
     - netcdf4
     - numpy >=2.0,<3.0
     - output_viewer ==1.3.3
+    - pcmdi_metrics >=3.9.3
     - xarray >=2023.02.0
     - xcdat >=0.7.3,<1.0
```

Comment:
I'm not sure if I can really speak to the other changes, but this part seems to match https://github.com/E3SM-Project/zppy-interfaces/blob/main/conda/dev.yml.

```
  - beautifulsoup4
  - lxml
  - matplotlib-base >=3.8.2
  - mpas_tools >=0.21.0
  - netcdf4
  - numpy >=2.0,<3.0
  - output_viewer=1.3.3
  - pcmdi_metrics>=3.9.3
  - xarray >=2023.02.0
  - xcdat >=0.7.3,<1.0
```